### PR TITLE
Vk stream handler done

### PR DESCRIPTION
**Pull request by romashok**

### CODE REVIEW

File: src/main/kotlin/org/mentionbattle/snadapter/impl/socialnetworks/handlers/vk/ClosableVkStreamingApiClient.kt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package org.mentionbattle.snadapter.impl.socialnetworks.handlers.vk
+
+import com.vk.api.sdk.client.TransportClient
+import com.vk.api.sdk.streaming.clients.VkStreamingApiClient
+
+class ClosableVkStreamingApiClient(transportClient: TransportClient?) : VkStreamingApiClient(transportClient) {
```
Comment (krasnotsvetov) [inline src/main/kotlin/org/mentionbattle/snadapter/impl/socialnetworks/handlers/vk/ClosableVkStreamingApiClient.kt]: I think, that we should remove this class and just use extension function. You can add it to Utils, in VkStreamingApiClientExtension.kt
[Docs](https://kotlinlang.org/docs/reference/extensions.html)

File: src/main/kotlin/org/mentionbattle/snadapter/impl/socialnetworks/handlers/VkHandler.kt [hunk 2]
```diff
@@ -6,29 +6,37 @@ import org.mentionbattle.snadapter.api.core.eventsystem.EventHandler
...
-import org.mentionbattle.snadapter.impl.socialnetworks.initalizers.VkServiceToken
+import org.mentionbattle.snadapter.impl.socialnetworks.initalizers.VkServiceAuth
 
-@SocialNetwork("VK")
-internal class VkHandler(token: VkServiceToken, tags : Tags, eventQueue : PrimitiveEventQueue)
+@SocialNetwork("vk")
+internal class VkHandler(auth: VkServiceAuth, tags: Tags, eventQueue: PrimitiveEventQueue)
     : SocialNetworkHandler, EventHandler {
 
     var isWorking = true
```
Comment (krasnotsvetov) [inline src/main/kotlin/org/mentionbattle/snadapter/impl/socialnetworks/handlers/VkHandler.kt]: Add private modifiers. Remove unnessary fields like isWorking
Did you move initialization with deleting rules from constuctor to process data?

File: src/main/kotlin/org/mentionbattle/snadapter/impl/socialnetworks/handlers/vk/objects/VkAccount.kt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+package org.mentionbattle.snadapter.impl.socialnetworks.handlers.vk.objects
+
+class VkAccount(val id: Int, val name: String, val avatarUrl: String)
```
Comment (krasnotsvetov) [inline src/main/kotlin/org/mentionbattle/snadapter/impl/socialnetworks/handlers/vk/objects/VkAccount.kt]: I think we should use here data class.
[Docs](https://kotlinlang.org/docs/reference/data-classes.html)

File: src/main/kotlin/org/mentionbattle/snadapter/impl/socialnetworks/handlers/vk/objects/RuleInfo.kt [hunk 4]
```diff
@@ -0,0 +1,3 @@
+package org.mentionbattle.snadapter.impl.socialnetworks.handlers.vk.objects
+
```
Comment (krasnotsvetov) [inline src/main/kotlin/org/mentionbattle/snadapter/impl/socialnetworks/handlers/vk/objects/RuleInfo.kt:2]: I think we should use here data class.
[Docs](https://kotlinlang.org/docs/reference/data-classes.html)

File: sna.config [hunk 5]
```diff
@@ -1,18 +1,19 @@
...
+             serviceToken : "put your service key here",
+             appId : put_your_application_id_key_here
           }
       },
       {
           Tags :
           {
-             contenderA : ["Санкт-Петербург", "Питер", "CПБ", "Saint-Petersburg"],
-             contenderB : ["Москва", "МСК", "Moscow"]
+             contenderA : ["Санкт-Петербург", "Питер", "CПБ", "Saint-Petersburg", "что", "где", "когда", "пожалуйста","спасибо" ],
```
Comment (krasnotsvetov) [inline sna.config]: Remove unnessary tags

**Review by krasnotsvetov — CHANGES_REQUESTED** [review]

File: src/main/kotlin/org/mentionbattle/snadapter/impl/socialnetworks/handlers/VkHandler.kt (hunk 2 above)
Comment (romashok) [reply src/main/kotlin/org/mentionbattle/snadapter/impl/socialnetworks/handlers/VkHandler.kt]: Yes. I also added "smart" (without unnecessary add and delete) validation that streaming rules are up to date.

**Comment by romashok** [comment]
done

**Pull request merged**